File: platform/sdk/python/examples/graph/graph-022_proj_label_in_1.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.projLabelIn1("node-multi", ["Employee", "Manager"])
if not result["ok"] or len(result["labelsApplied"]) != 2:
    sys.exit(1)
print("XJ-OK: graph-022_proj_label_in_1")
sys.exit(0)

File: platform/sdk/nodejs/examples/graph/graph-022_proj_label_in_1.js
const { GraphClient } = require("../../mox-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.projLabelIn1("node-multi", ["Employee", "Manager"]);
if (!result.ok || result.labelsApplied.length !== 2) process.exit(1);
console.log("XJ-OK: graph-022_proj_label_in_1");
process.exit(0);
